Describe the Form - Grouped Fields demo (#660)
DevExpress/devextreme-demos: JSDemos/Demos/Form/GroupedFields/description.md

> You can organize Form items in groups and tabs. To do this, declare [Group](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and [Tabbed](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/) items in the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array. Each group and tab can have their own layout and contain any other item types.

Reviewer: ```suggestion
You can organize Form items in groups and tabs. To do this, declare [Group](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and [Tabbed](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/) items in the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array. Each group and tab can have their own layout and contain other item types.
```

> You can organize Form items in groups and tabs. To do this, declare items of the [Group](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and [Tabbed](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/) types in the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array. Each group and tab can have their own layout and contain any other item types.

Reviewer 2: Just a small suggestion - maybe say "Group and Tabbed items" instead of "items of the Group and Tabbed types".

```suggestion
You can organize Form items in groups and tabs. To do this, declare [Group](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and [Tabbed](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/) items in the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array. Each group and tab can have their own layout and contain any other item types.
```

> To create a group item, assign "group" to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/#itemType) property. To specify a group's title, use the [caption](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/#caption) property. This demo shows three groups. The `Personal Data` group is nested in the `System Information` group.  

Reviewer 3: ```suggestion
To create a group item, assign *"group"* to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/#itemType) property. To specify a group's title, use the [caption](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/#caption) property. This demo shows three groups. The `Personal Data` group is nested in the `System Information` group.  
```

> The Form uses the [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) component to display tabs. To create a tabbed item, assign *"tabbed"* to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#itemType) property. This demo shows a tabbed item nested in the `Contact Information` group. You can specify the tab panel's settings in the [tabPanelOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#tabPanelOptions) object. This demo sets the [deferRendering](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#deferRendering) property to **false**.

Reviewer 3: I moved the first sentence closer to the information about tab panel settings - it seems more straightforward.

```suggestion
To create a tabbed item, assign *"tabbed"* to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#itemType) property. This demo shows a tabbed item nested in the `Contact Information` group. The Form uses the [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) component to display tabs. You can specify the tab panel's settings in the [tabPanelOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#tabPanelOptions) object. This demo sets the [deferRendering](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#deferRendering) property to **false**.
```

> The Form uses the [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) component to display tabs. To create a tabbed item, assign *"tabbed"* to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#itemType) property. This demo shows a tabbed item nested in the `Contact Information` group. You can specify the tab panel's settings in the [tabPanelOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#tabPanelOptions) object. This demo sets the [deferRendering](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#deferRendering) property to **false**.

Reviewer 3: > This demo sets the [deferRendering](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#deferRendering) property to **false**.

I think we need to clarify why it is so.

> To create a tabbed item, assign *"tabbed"* to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#itemType) property. This demo shows a tabbed item nested in the `Contact Information` group. The Form uses the [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) component to display tabs. You can specify the tab panel's settings in the [tabPanelOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#tabPanelOptions) object. This demo sets the [deferRendering](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#deferRendering) property to **false** to render TabPanel's content immediately.

Reviewer 3: Unifies the use of the terms TabPanel and properties and removes the repetition of "to" in the last sentence.

```suggestion
To create a tabbed item, assign *"tabbed"* to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#itemType) property. This demo shows a tabbed item nested in the `Contact Information` group. The Form uses the [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) component to display tabs. You can specify the TabPanel's properties in the [tabPanelOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#tabPanelOptions) object. This demo disables the [deferRendering](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#deferRendering) property to render TabPanel's content immediately.
```
